core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/event/DatabaseRestoreTriggerEvent.java

```diff
@@ -24,7 +24,7 @@ public DatabaseRestoreTriggerEvent(
             @JsonIgnoreDeserialization @JsonProperty("accepted") Promise<AcceptResult> accepted,
             @JsonProperty("backupLocation") String backupLocation,
             @JsonProperty("backupId") String backupId) {
-        super(event, resourceId, accepted, backupLocation, backupId);
+        super(event, resourceId, accepted, backupLocation, backupId, true);
```

Comment:
why was it necessary to include `true`? I think it is related to closing the connections but if so, it seems not to be necessary.

Reply:
Yes this is for closing connections. this was the only place the original constructor was used, so this a small refactor.